live-hls.js

```diff
@@ -673,6 +800,12 @@ dataRequest = function(request, response) {
   });
 };
 
+/**
```

Comment:
Don't have to do this now, but it might be wise to make the comments in this file more JSDoc-esque.